lib/fast_jsonapi/object_serializer.rb

```diff
@@ -156,7 +156,7 @@ def attributes(*attributes_list, &block)
         attributes_list.each do |attr_name|
           method_name = attr_name
           key = run_key_transform(method_name)
-          attributes_to_serialize[key] = AttributeSerializer.new(
+          attributes_to_serialize[key] = Attribute.new(
```

Comment:
Also, is there any chance that the instantiation of Attribute could slow things down? Obviously, the performance tests are passing, but what if there's tons of attributes? I'm ignorant to whether the instantiation/GC issues in AMS (issues that this library aims to solve) are related to actual class instantiation, or just how it serialized instantiated classes in general (in other words, whether using instantiated classes like this will affect performance at all).